Microsoft.NET.Build.Containers/Descriptor.cs

```diff
@@ -29,6 +29,15 @@ namespace Microsoft.NET.Build.Containers;
     [JsonPropertyName("digest")]
     public string Digest { get; init; }
 
+    /// <summary>
+    /// Digest of the uncompressed content, specifying algorithm and value.
+    /// </summary>
+    /// <remarks>
+    /// <see href="https://github.com/opencontainers/image-spec/blob/7b36cea86235157d78528944cb94c3323ee0905c/descriptor.md#digests"/>
+    /// </remarks>
+    [JsonIgnore]
+    public string UncompressedDigest { get; init; }
```

Comment:
Should this be a `string?` to avoid the new warnings?

(we'll turn on warnings as errors . . . eventually)

Reply:
Yes, it should be. I didn't even see the warnings... my eyes only see red I guess...

Reply:
> (we'll turn on warnings as errors . . . eventually)

💯